platform/spire/resources/servicemap.yaml

```diff
@@ -0,0 +1,41 @@
+services:
```

Comment:
What ensures that this file doesn't get out of sync with the actual set of services we run?

Reply:
The entries in the file won't get out of sync, because any deviation from the actual set of services that run will make things fail CI. However, nothing will catch us if we add additional services and forget to add them to this file, and I'm not sure exactly how we'd do that. Any ideas?

Reply:
Yes, I was talking about the latter scenario of course.  I don't have a good idea of how to solve this right now but we should keep it in mind